src/main/java/de/chojo/repbot/config/elements/PresenceSettings.java

```diff
@@ -0,0 +1,69 @@
+package de.chojo.repbot.config.elements;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import de.chojo.jdautil.localization.util.Replacement;
+import net.dv8tion.jda.api.entities.Activity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+@SuppressWarnings("FieldMayBeFinal")
+public class PresenceSettings {
+    private boolean active = false;
+    private int interval = 5;
+    private List<Presence> status = new ArrayList<>() {{
+        add(Presence.of(Activity.ActivityType.WATCHING, "%guild_count% guilds!"));
+        add(Presence.of(Activity.ActivityType.LISTENING, "%channel_count% channels!"));
+        add(Presence.of(Activity.ActivityType.WATCHING, "%total_rep% Reputations!"));
+        add(Presence.of(Activity.ActivityType.WATCHING, "%weekly_rep% Reputation this week!"));
+        add(Presence.of(Activity.ActivityType.WATCHING, "%today_rep% Reputation today!"));
+        add(Presence.of(Activity.ActivityType.WATCHING, "%weekly_avg_rep% Reputation per week!"));
+        add(Presence.of(Activity.ActivityType.LISTENING, "%analyzed_messages% messages during the last hour!"));
+    }};
```

Comment:
```suggestion
    private final boolean active = false;
    private final int interval = 5;
    private final List<Presence> status = List.of(
        Presence.of(Activity.ActivityType.WATCHING, "%guild_count% guilds!"),
        Presence.of(Activity.ActivityType.LISTENING, "%channel_count% channels!"),
        Presence.of(Activity.ActivityType.WATCHING, "%total_rep% Reputations!"),
        Presence.of(Activity.ActivityType.WATCHING, "%weekly_rep% Reputation this week!"),
        Presence.of(Activity.ActivityType.WATCHING, "%today_rep% Reputation today!"),
        Presence.of(Activity.ActivityType.WATCHING, "%weekly_avg_rep% Reputation per week!"),
        Presence.of(Activity.ActivityType.LISTENING, "%analyzed_messages% messages during the last hour!")
    );
```

Reply:
final fields can not be written by jackson.
I will use the List factory method ^^